components/add_user_to_channel_modal/add_user_to_channel_modal.tsx

```diff
@@ -24,9 +24,9 @@ import {RelationOneToOne} from '@mattermost/types/utilities';
 export type Props = {
 
     /**
-    * Function that's called when modal is closed
+    * Function that's called after modal is closed
     */
-    onHide: () => void;
+    onExited: () => void;
```

Comment:
This change needed to be made due how the `ModalController` works. The controller passes an `onHide` function that doesn't call the `onHide` handler passed by the parent, but it does for `onExited`. However we can't call both `onHide` and `onExited` passed by the controller as it bypasses `onExited`.